app/helpers/reactjs_helper.rb

```diff
@@ -1,5 +1,5 @@
 module ReactjsHelper
-  def mount_react_component(name, selector, data)
+  def mount_react_component(name, selector, data = [])
```

Comment:
@tbrisker yes, you are right that if we were using flash only kind of notifications, we could just pass the attributes, but that quickly breaks for JS generated events (e.g. dashboard or other users that use the notify function). as you would need to alter the existing notification state.